version.go

```diff
@@ -0,0 +1,34 @@
+package hcsshim
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/Microsoft/hcsshim/internal/hcs"
+	hcsschema "github.com/Microsoft/hcsshim/internal/schema2"
+)
+
+type SupportedSchemaVersionsStruct struct {

```

Comment:
>SupportedSchemaVersionsStruct [](http://example.com/codeflow?start=5&length=29)

This is part of schema, use name "BasicInformation"

Reply:
fix